tutorial.ipynb

```diff
@@ -0,0 +1,665 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load data\n",
+    "hit_trials = list()\n",
+    "data = list()\n",
+    "for id_idx, id in enumerate(subj_ids):\n",
+    "    fname = op.join(data_dir, 'prestim_humandetection_600hzMEG_subject' + id + '.mat')\n",
+    "    raw_data = loadmat(fname)\n",
+    "    hit_trials.append(np.nonzero(raw_data['YorN'].squeeze())[0])  # indices for trials where the stimulus was detected\n",
+    "    data.append(raw_data['prestim_raw_yes_no'])  # MEG time series (trials x samples)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once you've loaded the data, make sure you understand how it's formatted. There\n",
+    "are 10 subjects, each with 100 detected (i.e., *hit*) trials and 100 undetected (i.e., *miss*) trials. How might you go about selecting only the *hit* trials from the 1st subject?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(200, 600)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# investigate data structure: # trials x time for Subj 1\n",
+    "print(data[0].shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Identifying events in single-subject data\n",
+    "Let's find the spectral events across trial conditions in a single subject,\n",
+    "after which we'll expand the analysis to include all 10 subjects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "subj_data = data[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We've loaded the data and must now set the conditions governing Spectral Event detection:\n",
+    "- `freqs`, the frequency values over which you will calculate your **t**ime-**f**requency **r**esponse (TFR)\n",
+    "- `times`, the time values at which your signal was sampled relative to each epoch or trial\n",
+    "- `event_band`, the bounds of the frequency band in which you will look for Spectral Events\n",
+    "- `thresh_FOM`, the factor-of-the-median threshold that will be used to find suprathreshold spectral power in each frequency bin across the spectrogram(s)\n",
+    "\n",
+    "The general workflow for event detection follows this progression: **TIMESERIES** -> **TFR** -> **SPECTRAL EVENTS**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set parameters\n",
+    "freqs = list(range(1, 60 + 1))   # fequency values (Hz) over which to calculate TFR\n",
+    "times = np.arange(n_times) / samp_freq  # seconds\n",
+    "event_band = [15, 29]  # beta band (Hz)\n",
+    "thresh_FOM = 6.  # factor-of-the-median threshold"
```

Comment:
```suggestion
    "thresh_FOM = 6.0  # factor-of-the-median threshold"
```
Showing my non compsci colors :smile:

Reply:
Is there actually a difference, or is this merely a stylistic critique?

Reply:
No difference, I just think that removing the zeros only matters in source code where the character count matters for PEP8. Otherwise it just makes it hard to see what should genuinely be a scalar or an integer.